lib/sequel/adapters/drill.rb

```diff
@@ -80,6 +80,8 @@ def create_table_generator_class
 
     class Dataset < Sequel::Dataset
```

Comment:
Are you able to break these classes/modules up into their respective folders and files?